### PR TITLE
[Term Entry] Spyware

### LINKED ISSUE #2687: [Term Entry] Cybersecurity Malware spyware
### Reason/inspiration (optional)

We would like a new term entry for [Cybersecurity](https://www.codecademy.com/resources/docs/cybersecurity) about spyware.

The entry should go in a new file in `docs/content/cybersecurity/concepts/malware/terms/spyware/spyware.md`

The entry should: 

- Explain what spyware is.
- Explain why it's harmful.
- Explain how it infects a PC, and how to prevent it.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by CBID2**
### Description

I create a term entry about spyware for the Cybersecurity section
Fixes #2687 
### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [ ] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by CBID2** [comment]
Hey @KTom101. Can you help me out with this please?

File: content/cybersecurity/concepts/malware/terms/spyware/spyware.md
Comment (KTom101) [inline content/cybersecurity/concepts/malware/terms/spyware/spyware.md:3]: The current description is giving an error in YAML due to the apostrophes being picked up as closing single quotation marks. I provided a possible edit for this below:
```suggestion
Description: 'Spyware is malicious software designed to gather personal data from a device in order to send it to other third-party apps or companies without consent from the user.' 
```

File: content/cybersecurity/concepts/malware/terms/spyware/spyware.md
Comment (KTom101) [inline content/cybersecurity/concepts/malware/terms/spyware/spyware.md:11]: There should be a free course listed 
```suggestion
  - 'introduction-to-cybersecurity'
  - 'paths/cyber-resilience-risk-management'
```

File: content/cybersecurity/concepts/malware/terms/spyware/spyware.md
Comment (KTom101) [inline content/cybersecurity/concepts/malware/terms/spyware/spyware.md:15]:
```suggestion
 **Spyware** is malicious computer software designed to gather personal data from a device in order to send it to other third-party apps or companies without the consent or knowledge of the user.
```

File: content/cybersecurity/concepts/malware/terms/spyware/spyware.md
Comment (KTom101) [inline content/cybersecurity/concepts/malware/terms/spyware/spyware.md:17]: Title case headings (AP Style)
```suggestion
## Why Is Spyware Harmful?
```

File: content/cybersecurity/concepts/malware/terms/spyware/spyware.md
Comment (KTom101) [inline content/cybersecurity/concepts/malware/terms/spyware/spyware.md:21]:
```suggestion
## 	How Does Spyware Infect Devices?
```

File: content/cybersecurity/concepts/malware/terms/spyware/spyware.md
Comment (KTom101) [inline content/cybersecurity/concepts/malware/terms/spyware/spyware.md:29]:
```suggestion
## 	Signs of a Spyware-Infected Device
```

File: content/cybersecurity/concepts/malware/terms/spyware/spyware.md
Comment (KTom101) [inline content/cybersecurity/concepts/malware/terms/spyware/spyware.md:35]:
```suggestion
## Spyware Prevention Tips
```

File: content/cybersecurity/concepts/malware/terms/spyware/spyware.md
Comment (KTom101) [inline content/cybersecurity/concepts/malware/terms/spyware/spyware.md:39]:
```suggestion
- Avoid clicking on pop-up ads
```

**Review by KTom101 — CHANGES_REQUESTED** [review]
@CBID2 I've left some changes.

**Comment by CBID2** [comment]
Made the changes @KTom101. I’m not sure how to make the second test pass. Can you help me please?

**Review by KTom101 — APPROVED** [review]
@CBID2 This is ready for the next review. You can run `npm run format:all` to fix the two checks. It will format and lint at the same time. You can see more about this [here](https://www.codecademy.com/resources/docs/contribution-guide/getting-set-up-on-your-computer#:~:text=a%20new%20file.-,Additional%20set%20up%20needed,-There%20are%20continuous).

Also, you don't have to worry about it if you're having difficulty. We maintainers usually run the command for the author before we merge.

**Comment by CBID2** [comment]
> @CBID2 This is ready for the next review. You can run `npm run format:all` to fix the two checks. It will format and lint at the same time. You can see more about this [here](https://www.codecademy.com/resources/docs/contribution-guide/getting-set-up-on-your-computer#:~:text=a%20new%20file.-,Additional%20set%20up%20needed,-There%20are%20continuous).
> 
> Also, you don't have to worry about it if you're having difficulty. We maintainers usually run the command for the author before we merge.

Solved it @KTom101! :)

File: content/cybersecurity/concepts/malware/terms/spyware/spyware.md
Comment (SSwiniarski) [inline content/cybersecurity/concepts/malware/terms/spyware/spyware.md:20]:
```suggestion
Since Spyware steals a person's information and sends it others, it can put the person in all sorts of situations; whether it's receiving bills for unintended purchases, private information being leaked to the public, or preventing signing in to the user's Apple or Google Play account to download music and apps. In other words, spyware ruins lives.
```

File: content/cybersecurity/concepts/malware/terms/spyware/spyware.md
Comment (SSwiniarski) [inline content/cybersecurity/concepts/malware/terms/spyware/spyware.md:24]:
```suggestion
Spyware infects a person's devices in the following ways:
```

File: content/cybersecurity/concepts/malware/terms/spyware/spyware.md
Comment (SSwiniarski) [inline content/cybersecurity/concepts/malware/terms/spyware/spyware.md:42]:
```suggestion
- Avoid clicking on pop-up ads.
- Click on the trash icon when receiving emails that have recurring grammatical errors and containing messages where someone is asked to send personal information such as a bank account number or home address to solve a problem.
- Purchase reputable anti-virus software such as [Norton](https://us.norton.com/) or [McAfee](https://www.mcafee.com/).
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@CBID2 I did a review and left a couple of comments.

**Review by SSwiniarski — APPROVED** [review]
@CBID2 Looks good.

**Comment by CBID2** [comment]
> @CBID2 Looks good.

Thanks @SSwiniarski! :) By the way, I recently joined Code Crew :)

**Comment by SSwiniarski** [comment]
@CBID2 Congrats!

Live link here: https://www.codecademy.com/resources/docs/cybersecurity/malware/spyware

image

**Pull request merged**
